### ISSUE TITLE
`make install` failed because creationTimestamp is null

### ISSUE THREAD

**Issue by WindowsXp-Beta** [kubernetes-sigs/kubebuilder#3316]
### What broke? What's expected?

I'm new to kubebuilder and I followed the tutorial until [step1.9](https://book.kubebuilder.io/cronjob-tutorial/running.html). My `make install` failed with
```
error: unable to decode "STDIN": parsing time "null" as "2006-01-02T15:04:05Z07:00": cannot parse "null" as "2006"
```
After some searches, I found a [discussion thread](https://github.com/kubernetes-sigs/kubebuilder/discussions/2315#discussioncomment-5453266) and a related [issue](https://github.com/kubernetes-sigs/controller-tools/issues/402).
The solution I found is to delete this line: `creationTimestamp: null` in the generated CRD yaml every time I run `make manifests`. And since `make install` has `manifests` as its dependency, `make install` would overwrite my edit so I move this dependency.
So here is my question, is my workaround correct?
If so, maybe you should remind readers in the tutorial about this so they won't spend so much time googling like me😂.

### KubeBuilder (CLI) Version

Directly download the [project](https://github.com/kubernetes-sigs/kubebuilder/tree/master/docs/book/src/cronjob-tutorial/testdata/project)

### Plugin versions

```yaml
# Code generated by tool. DO NOT EDIT.
# This file is used to track the info used to scaffold your project
# and allow the plugins properly work.
# More info: https://book.kubebuilder.io/reference/project-config.html
domain: tutorial.kubebuilder.io
layout:
- go.kubebuilder.io/v4
projectName: project
repo: tutorial.kubebuilder.io/project
resources:
- api:
    crdVersion: v1
    namespaced: true
  controller: true
  domain: tutorial.kubebuilder.io
  group: batch
  kind: CronJob
  path: tutorial.kubebuilder.io/project/api/v1
  version: v1
  webhooks:
    defaulting: true
    validation: true
    webhookVersion: v1
version: "3"
```

### Other versions

```bash
./bin/kustomize version      
v5.0.0

./bin/controller-gen --version
Version: v0.11.3
```

### Extra Labels

/kind documentation

**Comment by killwing** [comment]
seems kustomize changes `creationTimestamp: null` to `creationTimestamp: "null"` when apply the patches.
delete the `creationTimestamp` should have no side effect.

**Comment by WindowsXp-Beta** [comment]
Thanks. Then it's not kubebuilder's fault. I'll close this issue.

**Comment by RafalSkolasinski** [comment]
Kind of have the feeling that in some sense it is kubebuilder's problem. The `creationTimestamp` should just not be there at all so maybe we should keep issue open until generated manifests stop containing it?

**Comment by jgillich** [comment]
FYI this was fixed in controller-tools v0.12. You can update the Makefile to get it:
```
CONTROLLER_TOOLS_VERSION ?= v0.12.0
```Not sure if this has undesired side effects in kubebuilder, but probably not
